lib/puppet/resource.rb

```diff
@@ -343,6 +343,7 @@ def initialize(type, title = nil, attributes = {})
         # set the type name to the symbolic name
         type = type.name
       end
+      @exported = false
```

Comment:
Actually, [it defaulted to false previously](https://github.com/puppetlabs/puppet/blob/master/lib/puppet/resource.rb#L101) too.